app/models/relationship.rb

```diff
@@ -0,0 +1,6 @@
+class Relationship < ApplicationRecord
+  belongs_to :follower, class_name: "User"
```

Comment:
```suggestion
  belongs_to :follower, class_name: User.name
```
Chúng ta nên dùng tên của bảng chứ không nên dùng String nhé em, mặc dù User.name vẫn trả ra String (yaoming)
em tìm hiểu xem vì sao nhé